CefSharp/Handler/IAccessibilityHandler.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright © 2010-2018 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+namespace CefSharp
+{
+    /// <summary>
+    /// Implement this interface to receive accessibility notification when accessibility events have been registered. 
+    /// The methods of this class will be called on the UI thread.
+    /// </summary>
+    public interface IAccessibilityHandler
+    {
+        /// <summary>
+        /// Called after renderer process sends accessibility location changes to the browser process.
+        /// </summary>
+        /// <param name="value">Updated location info.</param>
+        void OnAccessibilityLocationChange(object value);
```

Comment:
Does passing in `object` make sense here? Would it be better to create a wrapper around `CefValue`?

You can convert `CefValue` to `Json`, though not sure if that helps.

http://magpcss.org/ceforum/apidocs3/projects/(default)/(_globals).html#CefWriteJSON(CefRefPtr%3CCefValue%3E,cef_json_writer_options_t)

Reply:
Creating a wrapper is the next step I'm going to work on. For now I don't fully understand the structure on the objects being passed from CEF. All I have now is this code from CefClient
https://bitbucket.org/chromiumembedded/cef/src/a659b48fd1c041984cfaa0d335d60943bd2c6991/tests/cefclient/browser/osr_accessibility_helper.cc?at=master&fileviewer=file-view-default
but it is hard to tell what is the complete structure of there objects as it might not handle all the cases. Also they might change the structure in the future, so it might be easier to implement the parsing in C#.

Reply:
I know nothing about accessibility, so really cannot say. A wrapper around `CefValue` sounds reasonable.

Reply:
Sorry, I was thinking about something like `AccessibilityLocationChangeArgs` or `AccessibilityTreeChangeArgs`, this kind of wrapper. `CefValue` looks very much like `object`, it is a base class for various types: `int`, `bool`, `string`, `Dictionary`, `List` and others. So it does what `object` does. `TypeConversion.h` methods accept and return `object`:
`static CefRefPtr<CefValue> ToNative(Object^ value)`
`static Object^ FromNative(const CefRefPtr<CefValue>& value)`
I don't like passing `object` as a method argument, but it seems to make sense here.
What do you think?